sys-firmware/asahi-firmware/asahi-firmware-0.7.8.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{10..13} )
+
+inherit distutils-r1
+
+_name=asahi-installer
+
+DESCRIPTION="Asahi FW extraction script"
+HOMEPAGE="https://asahilinux.org"
+SRC_URI="https://github.com/AsahiLinux/${_name}/archive/refs/tags/v${PV}.tar.gz -> ${PN}-${PV}.tar.gz"
+S="${WORKDIR}/${_name}-${PV}"
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="arm64"
+
+DEPEND=">=sys-apps/asahi-scripts-20230606"
+RDEPEND="${DEPEND}
+	app-arch/lzfse
+	sys-kernel/linux-firmware
+"
+
+src_install() {
+	distutils-r1_src_install
+	keepdir /lib/firmware/vendor/
+}
+
+pkg_postinst() {
+	elog "Asahi vendor firmware update script"
+	elog "Please run 'asahi-fwupdate' after each update of this package."
+
+	if [ -e ${ROOT}/bin/update-vendor-fw -o -e ${ROOT}/etc/local.d/apple-firmware.start ]; then
```

Comment:
Bash tests, please: https://projects.gentoo.org/qa/policy-guide/ebuild-format.html#pg0101

Reply:
I'm just going to drop this anyway since no one should have either of these files anymore at this point.